<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaddleLesson</class>
 <widget class="QWidget" name="PaddleLesson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: black;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>91</y>
     <width>1279</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>1141</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OldNewspaperTypes</family>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="text">
      <string>The iambic paddle is an alternative device commonly used by morse code enthusiasts. Typically, while using a straight key, it is difficult to input correct morse above 18 words per minute. So, a new device was introduced that allows for much faster input, going up to even 30 words per minute.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="font">
     <font>
      <family>OldNewspaperTypes</family>
     </font>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>1141</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OldNewspaperTypes</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>The iambic paddle works by having two vertical paddle shaped levers that you can push. The left paddle is a dot, and the right paddle is a dash. Usually people make a sort of American Sign Language “g” character with their hand and move their hand right or left around the paddle to either input dots or dashes. In this app, the left arrow button is used as the left paddle, and the right arrow button is used as the right paddle.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>1141</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OldNewspaperTypes</family>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="text">
      <string>If you hold down one of the paddles, it will repeatedly emit either a dot or dash at the correct timings for you, based on your words per minute. If you hold both paddles down, it will alternate between sending dots and dashes.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>1141</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OldNewspaperTypes</family>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="text">
      <string>The best way to learn to use the paddle is to practice with it. It is best to first learn all of the morse characters with the straight key, and then move on to the paddle once you have the basics down, and you wish to improve how fast you can go. The paddle can also help you achieve more consistent timings as you input morse.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="headerFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-bottom: 2px solid black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>731</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old London</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lesson 10: The Paddle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;
background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>1281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-top: 2px solid black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="completeLessonButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>OldNewspaperTypes</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;
background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>Click To
Complete
Lesson</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextSlideButton">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;
background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="previousSlideButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;
background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
